libmproxy/console/flowdetailview.py

```diff
@@ -35,17 +35,20 @@ def flowtext(self):
         title = urwid.AttrWrap(title, "heading")
         text.append(title)
 
-        if self.flow.server_conn:
+        cc = self.flow.client_conn
+        sc = self.flow.server_conn
+        req = self.flow.request;
```

Comment:
so many languages in my coding life, made my fingers have code completion, but some times they mix things up ;)